# Description: revert commit which creates regressions in gtk
# Ubuntu: https://bugs.launchpad.net/freetype/+bug/972223
# Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=636776
--- a/src/truetype/ttdriver.c
+++ b/src/truetype/ttdriver.c
@@ -246,10 +246,7 @@
     FT_Request_Metrics( size->face, req );
 
     if ( FT_IS_SCALABLE( size->face ) )
-    {
       error = tt_size_reset( ttsize );
-      ttsize->root.metrics = ttsize->metrics;
-    }
 
     return error;
   }
